be/src/exprs/agg/java_window_function.cpp

```diff
@@ -0,0 +1,83 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#include "exprs/agg/java_window_function.h"
+
+#include <vector>
+
+#include "column/binary_column.h"
+#include "column/fixed_length_column.h"
+#include "column/nullable_column.h"
+#include "runtime/user_function_cache.h"
+
+namespace starrocks::vectorized {
+
+const AggregateFunction* getJavaWindowFunction() {
+    static JavaWindowFunction java_window_func;
```

Comment:
can the return value  be used accross threads?

Reply:
JavaWindowFunction does not hold any JNI objects or context